README.md

```diff
@@ -4,6 +4,12 @@
 
 The command-line interface for versum.
 
+- [versum-cli](#versum-cli)
```

Comment:
Started to add a list of documents that hopefully will help people (and us in the future) to keep track of everything.